src/query/api/v1/handler/influxdb/write_test.go

```diff
@@ -0,0 +1,49 @@
+/*
+ * Author: Markus Stenberg <mstenber@aiven.io>
```

Comment:
Again, may have to use the default headers... it's super annoying but blame the lawyers 🤦‍♂

Reply:
Not a problem for us, but in some jurisdictions (e.g. some EU countries IIRC) you cannot actually assign copyright from the original employee, just perma-license it onward.